reserve_harvester.py

```diff
@@ -95,10 +95,72 @@ def get_municipalities(title):
 
 def process_wp_reserves():
     """
+    Process a Petscan-generated list of nature reserves on svwp.
+
+    Attempt to find matches in the source file
+    and save them to appropriate report depending
+    on the reliability of the match (one match, multiple matches,
+    zero matches).
+    """
+    results_file_exact = "svwp_to_nature_id_exact.json"
+    results_file_none = "svwp_to_nature_id_none.json"
+    results_file_multiple = "svwp_to_nature_id_multiple.json"
+    results = []
+    results_none = []
+    results_multiple = []
+    reserves_on_wp = read_wp_nr_list()
+    article_count = len(reserves_on_wp)
+    reserves_source = read_reserve_csv()
+    print("Processing {} svwp articles.".format(article_count))
+    counter = 0
+    for article_title in reserves_on_wp:
+        if (not article_title.startswith("Lista") and
+                "nationalpark" not in article_title.lower()):
+            counter += 1
+            if counter % 10 == 0:
+                print("Processed {}/{}...".format(counter, article_count))
+            guesses = find_wp_reserve_in_data_file(
+                article_title, reserves_source)
+            if len(guesses) == 1:
+                entry = {}
+                entry["wp_article"] = article_title
+                entry["source_name"] = guesses[0]["name"]
+                entry["nature_id"] = guesses[0]["nature_id"]
+                entry["item"] = utils.q_from_wikipedia("sv", article_title)
+                results.append(entry)
+                utils.json_to_file(results_file_exact, results)
+            elif len(guesses) > 1:
+                entry = {}
+                entry["wp_article"] = article_title
+                nature_ids = []
+                for row in guesses:
+                    nature_ids.append(row["nature_id"])
+                entry["nature_id"] = nature_ids
+                results_multiple.append(entry)
+                utils.json_to_file(results_file_multiple, results_multiple)
```

Comment:
Wrt https://github.com/Vesihiisi/WLE_import/pull/2#discussion_r114568577

That is fine. Maybe just add a comment about it to explain.